app/templates/admin/createEvents.html

```diff
@@ -59,15 +59,19 @@ <h2> New Event Form</h2>
         </div>
         <div class="d-grid gap-1.5">
           <div class="form-group">
-            <label for='eventName'> <strong> Event Name * </strong></label> <br>
-            <input class="form-control"  id='inputEventName' value="{{eventInfo.eventName}}" name="eventName"
-                   {{'required' if user.isCeltsAdmin or user.isCeltsStudentStaff else 'readonly'}}>
```

Comment:
You're missing the 'required' attribute on all of these form elements now.

Reply:
We have added required back :)

Reply:
RSVP button functionality is fixed in PR #110